packages/appeals-service-api/__tests__/unit/controllers/local-planning-authorities.test.js

```diff
@@ -88,13 +48,11 @@ describe('LPAs controller test', () => {
 				name: filter
 			};
 			const data = ['some-filtered-array'];
-			LPA.find.mockResolvedValue(data);
+			service.getLpaList.mockResolvedValue(data);
```

Comment:
Same as above, we're not actually testing the filtering applied by the system, we're simply returning a value that we set-up to be returned